### PR TITLE
refactor: remove unused pedersen hasher

### PR DESCRIPTION
**Pull request by dignifiedquire**

### CODE REVIEW

File: fil-proofs-tooling/src/bin/benchy/prodbench.rs [hunk 1]
```diff
@@ -254,15 +254,11 @@ pub fn run(
...
-    // replica_inclusion (constraints: single merkle path pedersen)
     // data_inclusion (constraints: sha merklepath)
     // window_inclusion (constraints: merkle inclusion path in comm_c)
     // ticket_constraints - (skip)
-    // replica_inclusion (constraints: single merkle path pedersen)
-    // column_leaf_hash_constraints - (64 byte * stacked layers) pedersen_md
     // kdf_constraints
     // merkle_tree_datahash_constraints - sha2 constraints 64
-    // merkle_tree_hash_constraints - 64 byte pedersen
 }
```
Comment (porcuquine) [inline fil-proofs-tooling/src/bin/benchy/prodbench.rs:262]: Shouldn't these all be replaced with poseidon, rather than simply removed?

Comment (dignifiedquire) [reply fil-proofs-tooling/src/bin/benchy/prodbench.rs:262]: seems like these comments are not really used anymore anyway

**Review by porcuquine — APPROVED** [review]
- [x] Beware the Jubjub bird, and shun :tada:
- [ ]  The frumious Bandersnatch!

LGTM

File: fil-proofs-tooling/src/bin/benchy/prodbench.rs (hunk 1 above)
Comment (porcuquine) [reply fil-proofs-tooling/src/bin/benchy/prodbench.rs:262]: Maybe kill them all then?

**Review by cryptonemo — APPROVED** [review]

File: fil-proofs-tooling/src/bin/benchy/prodbench.rs (hunk 1 above)
Comment (dignifiedquire) [reply fil-proofs-tooling/src/bin/benchy/prodbench.rs:262]: done

**Pull request merged**